packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/AccessRequirementWikiInstructions.test.tsx

```diff
@@ -1,36 +1,41 @@
+import { ObjectType, WikiPage } from '@sage-bionetworks/synapse-types'
 import { render, screen, waitFor, within } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import React from 'react'
 import {
   mockACTAccessRequirement,
-  mockACTAccessRequirementWithWiki,
+  mockToUAccessRequirementWithWiki,
 } from '../../mocks/mockAccessRequirements'
-import { mockACTAccessRequirementWikiPage } from '../../mocks/mockWiki'
-import { server } from '../../mocks/msw/server'
+import { mockToUAccessRequirementWikiPage } from '../../mocks/mockWiki'
+import { rest, server } from '../../mocks/msw/server'
+import SynapseClient from '../../synapse-client'
+import {
+  confirmMarkdownSynapseTextContent,
+  expectMarkdownSynapseNotToGetWiki,
+  waitForMarkdownSynapseToGetWiki,
+} from '../../testutils/MarkdownSynapseUtils'
 import { createWrapper } from '../../testutils/TestingLibraryUtils'
-import { MarkdownSynapse } from '../Markdown'
-import { NO_WIKI_CONTENT } from '../WikiMarkdownEditorButton/WikiMarkdownEditorButton'
+import { WIKI_PAGE_ID } from '../../utils/APIConstants'
+import { BackendDestinationEnum, getEndpoint } from '../../utils/functions'
+import { NO_WIKI_CONTENT } from '../Markdown/MarkdownSynapse'
 import {
   AccessRequirementWikiInstructions,
   AccessRequirementWikiInstructionsProps,
 } from './AccessRequirementWikiInstructions'
 
-const MARKDOWN_SYNAPSE_TEST_ID = 'MarkdownSynapseContent'
-jest.mock('../Markdown/MarkdownSynapse', () => ({
-  __esModule: true,
-  default: jest.fn(),
-}))
-const mockMarkdownSynapse = jest.mocked(MarkdownSynapse)
-mockMarkdownSynapse.mockImplementation(
-  () => (<div data-testid={MARKDOWN_SYNAPSE_TEST_ID} />) as any,
-)
-async function confirmMarkdown(markdown: string) {
-  await screen.findByTestId(MARKDOWN_SYNAPSE_TEST_ID)
-  expect(mockMarkdownSynapse).toHaveBeenLastCalledWith(
-    expect.objectContaining({
-      markdown: markdown,
-    }),
-    expect.anything(),
+const createWikiPageSpy = jest.spyOn(SynapseClient, 'createWikiPage')
+const changeGetWikiPageHandlerOnce = (wikiPage: WikiPage) => {
```

Comment:
`MarkdownSynapse` will GET the WikiPage using `SynapseClient` rather than from the react query cache, so need to override the default getWikiPage handler to return the newly "created" WikiPage for `MarkdownSynapse` to GET